app/jinja_filters.py

```diff
@@ -534,27 +573,25 @@ def map_summary_item_config(
     ]
 
     if summary_type == "CalculatedSummary":
-        rows.append(
-            SummaryRow(calculated_question, summary_type, False, None, None, None)
-        )
+        rows.append(SummaryRow(calculated_question, summary_type, False, "", "", ""))
 
     return rows
 
 
 @blueprint.app_context_processor
-def map_summary_item_config_processor():
+def map_summary_item_config_processor() -> dict:
     return dict(map_summary_item_config=map_summary_item_config)
 
 
 @blueprint.app_template_filter()  # type: ignore
 def map_list_collector_config(
-    list_items,
-    icon,
-    edit_link_text=None,
-    edit_link_aria_label=None,
-    remove_link_text=None,
-    remove_link_aria_label=None,
-):
+    list_items: list,
+    icon: str,
+    edit_link_text: Optional[str] = "",
```

Comment:
These two `Optional[str]` have default value of `""` so they shouldn't be optional. Also `list_items: list` needs top level parameter.